deployment/gae/oss-vdb-test/app.yaml

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 service: default
-runtime: python311
+runtime: custom
+env: flex
```

Comment:
Are there any region / location changes as a result of this? 

Also, I assume our redis instances still work re networking ?